source/io/module_httpClient.js

```diff
@@ -44,8 +44,7 @@
 
         // WebVI Specific
         WEBVI_UNSUPPORTED_INPUT: 363798,
-        INVALID_HEADER: 363798,
-        FORBIDDEN_HEADER_WARNING: 363798
+        INVALID_HEADER: 363798
```

Comment:
Not a convention in our js code bases. Older browsers error on trailing commas. The linter enforces the right convention.